src/config/i18n/en/messages.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable */ module.exports = {
+  languageData: {
+    plurals: function(n, ord) {
+      var s = String(n).split('.'),
+        v0 = !s[1],
+        t0 = Number(s[0]) == n,
+        n10 = t0 && s[0].slice(-1),
+        n100 = t0 && s[0].slice(-2);
+      if (ord)
+        return n10 == 1 && n100 != 11
+          ? 'one'
+          : n10 == 2 && n100 != 12
+            ? 'two'
+            : n10 == 3 && n100 != 13
+              ? 'few'
+              : 'other';
+      return n == 1 && v0 ? 'one' : 'other';
+    },
+  },
+  messages: {
+    'common.close': 'Close',
+    'footer.legal.cookies': 'Cookies',
```

Comment:
what if we have it as an object for all? It will be easier to insert things there i.e.
```
messages: {
  footer: {
    legal: {  cookies: 'Cookies', privacy: 'Privacy and Policy  } etc
  }
}
```

Reply:
This file is generated, maybe we shouldn't store it in git. But I will have to add another task to the `start` task to compile the translations before starting webpack.

I think having it here is confusing, I will go ahead an remove it.

Reply:
Lingui doesn't allows to customize the keys in this way. We can use other format such as `po`: https://lingui.js.org/ref/conf.html#std:config-format

Reply:
Sounds good. As long as the file is not generated by us, it is fine 😄